sqlmesh/core/config/root.py

```diff
@@ -302,3 +303,20 @@ def dialect(self) -> t.Optional[str]:
     @property
     def fingerprint(self) -> str:
         return str(zlib.crc32(pickle.dumps(self.dict(exclude={"loader", "notification_targets"}))))
+
+
+def validate_no_past_ttl(v: str) -> str:
```

Comment:
Can we stick to an existing pattern instead? I'm not sure I like the inconsistency

Reply:
Updated the other field validators: [`900b163` (#4158)](https://github.com/TobikoData/sqlmesh/pull/4158/commits/900b163dfb32e918c43c53407e5d659ba435a375)